docs/sources/tempo/setup/operator/object-storage.md

```diff
@@ -17,9 +17,7 @@ Tempo Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure
 
 * Create a [bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) on AWS.
 
-### Installation
-
-* Deploy the Tempo Operator to your cluster.
+### Static Token Installation
```

Comment:
Changing the case to match our Writer's Toolkit. We use [sentence case](https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/) instead of title case in our documentation. 

```suggestion
### Static token installation
```